src/chains/algorand_1/algoTransaction.ts

```diff
@@ -228,7 +212,10 @@ export class AlgorandTransaction implements Transaction {
   /** Add signatures to raw transaction
    */
   addSignatures = (signatures: AlgorandSignature[]): void => {
-    this._signatures = signatures
+    signatures.forEach(signature => {
+      isValidAlgorandSignature(signature)
```

Comment:
I'm sure you meant to use a different function here. Look at the EOS addSignatures function

Reply:
done